hub/apps/how-tos/uno-multiplatform.md

```diff
@@ -0,0 +1,151 @@
+---
+title: How to target multiple platforms with your WinUI 3 app
+description: Reach more users with a single WinUI 3 / .NET codebase using Uno Platform.
+ms.topic: article
+ms.date: 3/14/2023
+keywords: uno platform, winui
+ms.author: aashcraft
+author: alvinashcraft
+ms.localizationpriority: medium
+ms.custom: template-quickstart
+audience: new-desktop-app-developers
+content-type: how-to
+---
+
+# How to target multiple platforms with your WinUI 3 app
+
+Once you've [created](/hub/apps/how-tos/hello-world-winui3.md) a starter Hello World WinUI 3 app, you might be wondering how to reach more users with a single codebase. This how-to will use Uno Platform to expand the reach of your existing application enabling reuse of the business logic and UI layer across native mobile, web, and desktop.
+
+:::image type="content" source="images/hello-world/uno-hello-world.png" alt-text="The 'Hello world' app running in the browser.":::
+
+## Prerequisites
+
+- [Visual Studio 2022 17.4 or later](https://visualstudio.microsoft.com/#vs-section)
+- [Tools for Windows App SDK](../windows-app-sdk/set-up-your-development-environment.md)
+- ASP.NET and web development workload (for WebAssembly development)
+:::image type="content" source="images/hello-world/vs-install-web.png" alt-text="Web development workload in VS":::
+- .NET Multi-platform App UI development installed (for iOS, Android, Mac Catalyst development).
+:::image type="content" source="images/hello-world/vs-install-dotnet-mobile.png" alt-text="dotnet mobile workload in VS":::
+- .NET desktop development installed (for Gtk, Wpf, and Linux Framebuffer development)
+:::image type="content" source="images/hello-world/vs-install-dotnet.png" alt-text=".net desktop workload in VS":::
+
+## Finalize your environment
+
+1. Open a command-line prompt, Windows Terminal if you have it installed, or else Command Prompt or Windows Powershell from the Start Menu.
+
+2. Install the `uno-check` tool:
+    - Use the following command:
+
+        `dotnet tool install -g uno.check`
+
+    - To update the tool, if you already have an existing one:
+
+        `dotnet tool update -g uno.check`
+
+3. Run the tool with the following command:
+
+    `uno-check`
+
+4. Follow the instructions indicated by the tool. Because it needs to modify your system, you may be prompted for elevated permissions.
+
+## Install the Uno Platform solution templates
+
+Launch Visual Studio 2022, then click `Continue without code`. Click `Extensions` -> `Manage Extensions` from the Menu Bar.
```

Comment:
Try to avoid specifying the Visual Studio version anywhere other than the pre-requisites section. It's more content to update later when new versions are released.
```suggestion
Launch Visual Studio, then click `Continue without code`. Click `Extensions` -> `Manage Extensions` from the Menu Bar.
```